Directions Example/MBDrawingView.swift

```diff
@@ -0,0 +1,100 @@
+import UIKit
+
+public protocol MBDrawingViewDelegate: class {
+    func drawingView(drawingView: MBDrawingView, didDrawWithPoints points: [CGPoint])
+}
+
+public class MBDrawingView: UIView {
+    
+    private var points: [CGPoint]!
+    private var context: CGContext!
+    private var strokeColor = UIColor.blue.withAlphaComponent(0.75)
+    private var lineWidth: CGFloat = 3
+    
+    public weak var delegate: MBDrawingViewDelegate?
+    
+    public convenience init(frame: CGRect, strokeColor: UIColor, lineWidth: CGFloat) {
+        self.init(frame: frame)
+        
+        self.strokeColor = strokeColor
+        self.lineWidth = lineWidth
+    }
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+    
+    deinit {
+        UIGraphicsEndImageContext()
+    }
+    
+    public func setStrokeColor(strokeColor: UIColor) {
```

Comment:
Why manual setters here and not in `strokeColor`'s `didSet`? Especially considering that these are `public`, I would argue that `setThing(_ thing: ThingType)` "setter" methods are an anti-pattern.